Sources/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -10,6 +10,7 @@ import Foundation
 /// `ChatChannelController` objects are lightweight, and they can be used for both, continuous data change observations (like
 /// getting new messages in the channel), and for quick channel mutations (like adding a member to a channel).
 ///
+/// - Note: Please refer to ``Chat`` in the async-await supported [state layer](https://getstream.io/chat/docs/sdk/ios/client/state-layer/state-layer-overview/).
```

Comment:
Nice way to promote the state layer. I would rephrase these a bit though - something like "For an async-await alternative of the $ClassName, please check ..."